src/CommunityToolkit.Maui/Views/LazyView/LazyView.shared.cs

```diff
@@ -1,28 +1,60 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace CommunityToolkit.Maui.Views;
+﻿namespace CommunityToolkit.Maui.Views;
 
 /// <summary>
-/// This a basic implementation of the LazyView based on <see cref="BaseLazyView"/> use this an example to create yours
+/// This a basic implementation implementing <see cref="LazyView"/>
 /// </summary>
 /// <typeparam name="TView">Any <see cref="View"/></typeparam>
-public class LazyView<TView> : BaseLazyView where TView : View, new()
+public class LazyView<TView> : LazyView where TView : View, new()
 {
 	/// <summary>
-	/// This method initializes your <see cref="LazyView{TView}"/>.
+	/// This method initializes <see cref="LazyView{TView}"/>.
 	/// </summary>
 	/// <returns><see cref="ValueTask"/></returns>
 	public override ValueTask LoadViewAsync()
 	{
-		View view = new TView { BindingContext = BindingContext };
+		Content = new TView { BindingContext = BindingContext };
 
-		Content = view;
+		SetHasLazyViewLoadedLoaded(true);
```

Comment:
Just to confirm... We don't want this extra `Loaded`, right?
```suggestion
		SetHasLazyViewLoaded(true);
```

Reply:
Good catch! Correct 💯 

That was a ctl + F error

Reply:
LoadedLoaded - to be sure to be sure ;-)

Thanks for the fixes!